conode/run_conode.sh

```diff
@@ -135,7 +135,7 @@ runLocal(){
 		fi
 
 		if [ ! -d $co ]; then
-			echo -e "localhost:$((7000 + 2 * $n))\nConode_$n\n$co" | conode setup
+			echo -e "localhost:$((2000 + 2 * $n))\nConode_$n\n$co" | conode setup
```

Comment:
use `7000` again. For running it on the server, run `run_conode.sh` once, then replace all 700x with 200x:

```
perl -pi -e "s/700/200/" *.toml co?/*.toml
```